src/styles/header.scss

```diff
@@ -75,6 +75,7 @@ $divider-color: rgba(0, 0, 0, 0.2);
     right: 0;
     top: $icon-size;
     width: 200px;
+    z-index: 1;
```

Comment:
`z-index` can get really hard to track due to being magic values almost by definition, so it might be good to add a comment here referring to the element you want this to overlap, and the z-index it has at the time of writing (the default), in case someone wants to overlap the menu again in the future, or the menu gets overlapped by something else.

Reply:
I'll push this as a simple commit to `master`